proposals/NNNN-resource-instance-analysis.md

```diff
@@ -0,0 +1,71 @@
+* Proposal: [NNNN - Resource Instance Analysis](NNNN-resource-instance-analysis.md)
+* Author(s): [Ashley Coleman](https://github.com/V-FEXrt)
+* Status: **Design In Progress**
+
+## Introduction
+
+This proposal introduces a new term, Resource Instance, which represents a 
+unique and specific resource. The instance is referenced via a handle returned
+from the create handle intrinsics and it is possible to have multiple handles
+pointing at a specific Resource Instance, but each unique instance is
+disambiguated by the handle creation parameters.
+
+Beyond creation parameters, Reource Instances have several pieces of associated
+metadata. Type, CounterDirection (HasCounter), and GloballyCoherent are
+identified examples. Type is currently handled well so this proposal suggests
+no changes. However CounterDirection and GloballyCoherent require careful late 
+stage analysis to derive the correct metadata.
+
+To serve the needs of CounterDirection and to setup a framework for future
+instance metadata this document proposes that instance analysis be combined
+into the DXILResourceBindingAnalysis pass and that the pass be renamed.
+
+
+## Proposed solution
+
+In order to support the generic resource instance analysis `DXILResourceBindingAnalysis` 
+and `ResourceBindingInfo` will be renamed to `DXILResourceAnalysis` and `ResourceInfo`
+respectively. Then associated instance metadata will be resolved during the
+`DXILResourceBindingAnalysis` pass.
```

Comment:
```suggestion
`DXILResourceAnalysis` pass.
```
This pass has been renamed?  Or are there going to be both a DXILResourceBindingAnalysis pass and a DXILResourceAnalysis pass?

Reply:
renamed!